### PR TITLE
chore(bithound): update .bithoundrc

### PR DESCRIPTION
**Pull request by zydc2010**

### CODE REVIEW

File: .bithoundrc [hunk 1]
```diff
@@ -1,7 +1,22 @@
 {
+  "ignore": [
+    "**/node_modules/**",
+    "**/docs/**",
+    "**/custom_typings/*.d.ts"
```
Comment (effervescentia) [inline .bithoundrc]: can skip everything inside `custom_typings`

Comment (zydc2010) [inline .bithoundrc]: Only *.d.ts files inside

Comment (effervescentia) [inline .bithoundrc]: so? we don't want to scan anything in there no matter what the file extension

Comment (zydc2010) [inline .bithoundrc]: Do I need to change to `**` instead?

**Comment by zydc2010** [comment]
Done done done

**Pull request merged**